.circleci/config.yml

```diff
@@ -74,11 +80,18 @@ workflows:
           requires:
             - publish-dev-sha
 
-  test-integration_publish-to-registry:
+  test-integration:
     when: << pipeline.parameters.run-integration-tests >>
     jobs:
       - test-integration-exec
       - test-integration-other-orb
+
+  publish-to-registry:
+    unless: << pipeline.parameters.run-integration-tests >>
+    jobs:
+      - approve:
+          type: approval
```

Comment:
Yeah, we don't _need_ it. It's just that the CircleCI config itself doesn't enforce anything right now. All the responsibility is on GitHub now, so that's why I thought I'd add that little extra layer. Should I remove it?

Reply:
Or leave it in for this release and remove it when we're more confident about the process?

Reply:
I'm fine with leaving it in for now and removing it later. In the final state, I'd like to have as little manual steps as possible.